persistence/in-memory/index_gatewaydedupe_core_[7.2,).partial.md

```diff
@@ -1,5 +1,7 @@
 ## Gateway deduplication
 
+partial: gatewayinmemory
```

Comment:
this approach shouldn't require two different versions when the targeted partial is already versioned?

Reply:
ok wow this is complicated because I assume the intention is to point towards the gateway 3.1 API once using core v7. I think it might be easier to move all of this into the dedicated gateway page but that's out of scope for this PR.

Reply:
Fixed it, nested partials are not supported. Now using 3 partials

- Prior to 7
- 7 till 7.2
- After 7